assets/app/scripts/services/auth.js

```diff
@@ -276,4 +276,16 @@ angular.module('openshiftConsole')
       }
     }
   };
-}]);
+}])
+
+/* A WebSocket factory for kubernetesContainerTerminal */
+.factory("AuthContainerWebSocket", function(API_CFG, $ws) {
```

Comment:
because we're getting auth by virtue of the registered interceptors now, this is orthogonal to auth... move to ws.js or to its own file?